dna/__main__.py

```diff
@@ -121,6 +121,9 @@ def configure_evaluate_parser(parser):
         '--no-cache', default=False, action='store_true',
         help='when set, do not use cached preprocessed data'
     )
+    parser.add_argument(
+        '--metafeature-subset', type=str, default='all', choices=['all', 'just-landmarks', 'no-landmarks']
```

Comment:
The term in the literature is "landmarker". Let's call the choices 'landmarkers' and 'no-landmarkers'.